include/simdjson/inline/error.h

```diff
@@ -1,6 +1,7 @@
 #ifndef SIMDJSON_INLINE_ERROR_H
 #define SIMDJSON_INLINE_ERROR_H
 
+#include <cstring>
```

Comment:
It is probably best to have all system includes regrouped, so this include should either be moved two lines down or the `include <string>` should be moved up.